internal/controllers/inventory_controller.go

```diff
@@ -1283,14 +1429,14 @@ func (t *reconcilerTask) createIngress(ctx context.Context) error {
 								},
 							},
 							{
-								Path: "/o2ims-infrastructureMonitoring/v1/alarmSubscriptions",
+								Path: "/o2ims-infrastructureMonitoring",
```

Comment:
No, it will directly everything starting with this path prefix to the alarm server.

Reply:
it's been a while since I last worked with ingress, but if I remember correctly, when a path matches multiple rules, it uses the more specific one. Wouldn't be have problems with "/o2ims-infrastructureMonitoring" and "/". It will match both, but everything will be sent to the alarm backend. Not sure, will look it up as well.

Reply:
I misread `o2ims-infrastructureMonitoring` - ignore